src/app/components/EditionPicker.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react'
+import { Button } from 'react-bootstrap'
+import Speakers from './Speakers.js'
+import Companies from './Companies.js'
+import Team from './Team.js'
+
+class EditionPicker extends Component {
+  constructor () {
+    super()
+    this.state = {
+      edition: 'sinfo24'
+    }
+  }
+
+  render () {
+    return (
+      <div className='Edition-picker'>
+        <Button onClick={() => this.setState({edition: 'sinfo21'})}>SINFO 21</Button>
```

Comment:
I think just for the purposes used here (examples) we can leave it as it is, there's no point in over engineering this.